.appveyor.yml

```diff
@@ -0,0 +1,59 @@
+cache:
+  - c:\tools\vcpkg\installed\
+
+configuration: Release
+
+image:
+  - Visual Studio 2015
+  - Visual Studio 2017
+
+platform:
+  - x64
+  - x86
+
+environment:
+  matrix:
+    - generator: "Visual Studio 14 2015"
+    - generator: "Visual Studio 15 2017"
+
+
+matrix:
+  exclude:
+    - image: "Visual Studio 2015"
+      generator: "Visual Studio 15 2017"
+    - image: "Visual Studio 2017"
+      generator: "Visual Studio 14 2015"
```

Comment:
I think that combination (image and generator) is wrong.
```patch
    - image: "Visual Studio 2015"
-     generator: "Visual Studio 15 2017"
+     generator: "Visual Studio 14 2015"
    - image: "Visual Studio 2017"
-     generator: "Visual Studio 14 2015"
+     generator: "Visual Studio 15 2017"
```

Reply:
That's the exclusion matrix. I'm excluding that combination from the tests it runs.

Reply:
Sorry, I had misunderstood.